api/envoy/api/v2/core/backoff.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.api.v2.core;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.api.v2.core";
+option java_outer_classname = "BackoffProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.config.core.v3";
+
+// [#protodoc-title: Backoff Strategy]
+
+// Configuration defining a jittered exponential back off strategy.
+message BackoffStrategy {
+  // The base interval to be used for the next back off computation. It should
+  // be greater than zero and less than or equal to :ref:`max_interval
+  // <envoy_api_field_core.BackoffStrategy.max_interval>`.
+  google.protobuf.Duration base_interval = 1 [(validate.rules).duration = {gt {}}];
+
+  // The cap on the next back off value.
+  google.protobuf.Duration max_interval = 2 [(validate.rules).duration = {gt {}}];
```

Comment:
I'm pretty sure that unless you set the fields to required, the > 0 is not actually enforced (see other duratoin examples). I think you want both of these fields to be required inside the message? Otherwise I would clearly document the default values?

Reply:
Make `base_interval ` field as required and add document for `max_interval`